guides/common/attributes-base.adoc

```diff
@@ -73,6 +73,10 @@
 :package-remove: yum remove
 :package-update-project: yum update
 :package-update: yum update
+:postgresql-lib-dir: /var/lib/pgsql
+:postgresql-data-dir: {postgresql-lib-dir}/data
+:postgresql-conf-dir: {postgresql-data-dir}
+:postgresql-log-dir: {postgresql-data-dir}/log
```

Comment:
These are the EL8 values. For Satellite 6.12 this should be safe since it only supports RHEL 8, which technically means it's also a bugfix in various places.

For upstream it's a bit unclear since both EL7 & EL8 should be supported, but that's now EOL anyway so I'm not too worried.

For orcharhino I don't know what it means.

Reply:
Since there is no orcharhino using Foreman 3.3 yet, I think this is fine! I will create a follow-up PR in case orcharhino differs.